src/components/speakerCard/SpeakerCard.css

```diff
@@ -0,0 +1,77 @@
+.codeutsava__section4-speaker-card{
+    background: #FFF9F9;
+    box-shadow: 0px 3px 5px 3px rgba(0, 0, 0, 0.25);
+    border-radius: 16px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    padding: 2rem 2rem 0.5rem 2rem;
+    max-width: 370px;
+}
+
+.codeutsava__section4-speaker-menu{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+
+.codeutsava__section4-speaker-menu h1{
+    font-weight: 700;
+    font-size: 27px;
+    line-height: 33px;
+    text-transform: uppercase;
+    text-align: center;
+    color: black;
+    padding-top: 1rem;
+}
+
+.codeutsava__section4-speaker-menu h3{
+    font-size: 20px;
+    line-height: 20px;
+    color: #004AAD;
+    padding-top: 0.5rem;
+}
+
+.codeutsava__section4-speaker-menu p{
+    font-weight: 300;
+    font-size: 15px;
+    line-height: 18px;
+    text-align: center;
+    padding: 1rem 0;
+}
```

Comment:
Use roboto font for paragraphs

Reply:
Added robot font in paragraphs in speaker card and two more cards to show vertical positioning of cards.